server/game/gamesteps/decksearchprompt.js

```diff
@@ -67,8 +67,8 @@ class DeckSearchPrompt extends UiPrompt {
     }
 
     buttons() {
-        let uniqueCardsByTitle = _.uniq(this.searchCards(), card => card.cardData.label);
-        let buttons = _.map(uniqueCardsByTitle, card => {
+        this.uniqueCardsByTitle = _.uniq(this.searchCards(), card => card.cardData.label);
```

Comment:
it was a toss up between putting this and 'buttons' on the class, but since there's a method already called buttons(), this won